projects/pastanaga-angular/src/lib/popup/popup.directive.ts

```diff
@@ -82,7 +79,31 @@ export class PopupDirective implements OnInit {
         }
     }
 
-    getPosition(override?: PositionStyle, contextualEvent?: MouseEvent | false): PositionStyle {
+    @HostListener('mouseenter', ['$event'])
+    onHover($event: MouseEvent) {
+        if (this._openOnHover && !this._disabled && !this.paPopup?._isDisplayed) {
+            this.paPopup?.show(this.getPosition($event));
+        }
+    }
+    @HostListener('mouseleave')
+    onLeave() {
+        if (this._openOnHover && !this._disabled && !!this.paPopup?._isDisplayed) {
+            this.paPopup.close();
+        }
+    }
+
+    private getPosition($event: MouseEvent) {
```

Comment:
we could probably merge these two methods now

Reply:
indeed, sorry I pushed my PR and realised I forgot that right after so I force pushed the fix before seeing your comment